aten/src/ATen/native/kleidiai/kai_ukernel_interface.h

```diff
@@ -0,0 +1,145 @@
+// Copyright 2024 Arm Ltd. and/or its affiliates
+
+#pragma once
+#include <ATen/Config.h>
+#include <unordered_map>
+#if AT_KLEIDIAI_ENABLED()
+
+#include <kai_lhs_quant_pack_qai8dxp_f32.h>
```

Comment:
why can't these *_interface.cpp and *_interface.h be defined inside the kleidi repo?
these function definitions have nothing to do with torch/aten. 
Moving these to kleidi repo provides a clean interface, which makes its integration into pytorch cleaner,  and also makes it usable in other frameworks.

Reply:
> ML Frameworks need to have adapter to connect with those interfaces as each framework has different API requirements.

But I don't see any PyTorch/aten specifics in any of these interface functions!